cit-api/pipeline/management/commands/bootstrap.py

```diff
@@ -17,39 +19,25 @@ def handle(self, *args, **options):
         # print("Importing list of data sources...")
         # import_data_sources()
 
-        # import census subdivisions first because a lot of other tables have foreign keys to
-        # CensusSubdivision
-        print("Importing CensusSubdivisions...")
-        import_shp_resources("census")
-
-        print("Importing Roads...")
-        import_shp_resources("roads")
-
-        print("Importing connectivity Hexes...")
-        import_shp_resources("hexes")
-
-        # import remaining shp resources
-        shp_resources = DataSource.objects.filter(source_type="shp").exclude(name__in=["census", "roads", "hexes"])
-        for resource in shp_resources:
-            print("Importing {}...".format(resource.name))
-            import_shp_resources(resource.name)
-
-        # import communities first because many tables foreign key to Community
-        print("Importing Communities...")
-        import_csv_resources("communities")
-
-        print("Importing Services...")
-        import_csv_resources("services")
-
-        # import remaining csv resources
-        csv_resources = DataSource.objects.filter(source_type="csv").exclude(name__in=["communities", "services"])
-        for resource in csv_resources:
-            print("Importing {}...".format(resource.name))
-            import_csv_resources(resource.name)
-
-        print("Importing databc locations...")
-        import_databc_resources("all")
-        calculate_nearest_location_types_outside_50k()
+        # import remaining resources
+
+        resources = DataSource.objects.order_by('import_order')
```

Comment:
Maybe I missed the sets somewhere. Where is the `import_order` being set? I assume we want to have part of the migration set this for consistency.

Reply:
I forgot to add the comment that I will share the new and updated data sources list

Reply:
Awesome! That's my only concern here.